### DISCUSSION TITLE
Wp-GraphQL Doen't Work with Other Auth Libraries

### DISCUSSION THREAD

**Discussion by creatzor** [wp-graphql/wp-graphql#3026] [General] [answered]
Hey folks,

I'm using https://wordpress.org/plugins/simple-jwt-login/ to handle JWT auth for REST endpoints (the application utilizes both authenticated REST and GraphQL). The token generated by that plugin doesnt work for wp-graphql. I consistently receive `null` when querying `viewer`.

I pass the token through the `Authorization` header. REST works, but graphql doesnt. Can someone clarify whether I have to use the bespoke wp-graphql auth plugin or not? 

Any help would be deeply appreciated. Thank you!

**Comment by justlevine** [comment, marked as answer]
@creatzor , there are other auth plugins for WPGraphQL  ( e.g. https://GitHub.com/AxeWP/wp-graphql-headless-login), but yea a plugin that works on a REST endpoint has no bearing on GraphQL - unless the plugin actively supports that too.

**Reply by creatzor** [reply]
Got it, thank you!
